config-linux.md

```diff
@@ -384,6 +384,31 @@ The following parameters can be specified to set up the controller:
     }
 ```
 
+## <a name="configLinuxVTPMs" />vTPMs
+
+**`vtpms`** (array of objects, OPTIONAL) lists a number of emulated TPMs that
+will be made available to the container.
+
+Each entry has the following structure:
+
+* **`statePath`** *(string, REQUIRED)* - full path to a directory where the vTPM is to write its persistent state into
+* **`vtpmVersion`** *(string, OPTIONAL)* - The version of TPM to emulate; either 1.2 or 2; default is 1.2
+* **`createCerts`** *(boolean, OPTIONAL)* - Whether to create certificates for the vTPM
```

Comment:
I think we also want to say “Defaults to `false`.”, although existing uses ([here][1] and [here][2]) have tacked that onto the first sentence-ish and not backticked the JSON literal.

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config.md#L45
[2]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config.md#L148

Reply:
I reformulated that a bit to "If true then create certificates for the vTPM, default to false."